decidim-accountability/app/packs/src/decidim/accountability/admin/imports.js

```diff
@@ -0,0 +1,16 @@
+const tuggleInfo = (val) => {
+  document.querySelectorAll(".help-text").forEach((toBeHidden) => {
```

Comment:
This seems a pretty complex implementation if we already know the ID based on `val`.

Why not just target the element using its ID?

Reply:
We need to hide all of the elements inside the "help-text" class, and only display the val(some other options might be displayed from the previous user selection, so we need to go through all of the elements)

Reply:
OK, thanks for the explanation!